src/index.ts

```diff
@@ -1,9 +1,18 @@
 import { install } from "./stages/install";
 import { configure } from "./stages/configure";
+import { push } from "./stages/push.ts";
+import { getState, saveState } from "@actions/core";
+
+const isPost = !!getState("isPost");
```

Comment:
unnecessary

Reply:
can typescript really automatically convert from string to boolean? 🤔